src/api-service/__app__/onefuzzlib/events.py

```diff
@@ -33,9 +50,54 @@ def get_events() -> Optional[str]:
         return None
 
 
+def log_event(event: Event, event_type: EventType) -> Event:
+    
+    clone_event = event.copy(deep=True)
+    log_event_recurs(clone_event)
+    logging.info("sending event: %s - %s", event_type, clone_event)
+    
+    return clone_event
+
```

Comment:
```suggestion

def filter_event(event: Event) -> Event:
    clone_event = event.copy(deep=True)
    return filter_event_recurse(event)


```

Reply:
Where are you adding this? If you're just updating the log_event function, it needs to actually 'log'. That's why we want to run the function. Maybe I'm just not reading this correctly.